src/AboutContent.js

```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
     <wrapper>
```

Comment:
In the AboutPage (above this component level), I'd suggest we center the content box in the row. The width looks reasonable, but the fact that it's left aligned in a larger container is a bit odd.